src/refactorings/add-braces-to-if-statement/index.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  addBracesToIfStatement,
+  hasIfStatementToAddBraces
+} from "./add-braces-to-if-statement";
+
+import { RefactoringWithActionProvider } from "../../types";
+
+const config: RefactoringWithActionProvider = {
+  command: {
+    key: "addBracesToIfStatement",
+    operation: addBracesToIfStatement,
+    title: "Add Braces To If Statement"
+  },
+  actionProvider: {
+    message: "Add braces to if statement",
+    canPerform: hasIfStatementToAddBraces
+  }
+};
+
+export default config;
```

Comment:
All of this is very nice. The essentials of the feature seems to be here. We need to declare the refactoring in the `extension.ts` file and in the `package.json`, like the others, so it becomes available. 

You should be able to see it if you run the extension host debugger (`F5`).

Reply:
Thank you, I will have a look to those 2 files.